runtime/compiler/ilgen/Walker.cpp

```diff
@@ -6719,6 +6719,38 @@ TR_J9ByteCodeIlGenerator::genAconst_init(TR_OpaqueClassBlock *valueTypeClass, in
    genFlush(0);
    }
 
+/**
+ * Check the call chain for the following pattern: StringBuilder.toString() -> String<init>(StringBuilder)
+ *                -> String<init>(AbstractStringBuilder, Void) -> Arrays.copyOfRange([BII)
+ *
+ * @param comp the compilation class.
+ * @return true if the call chain matches the expected stack or false otherwise.
+ */
+static bool isCallChainFromStringBuilderDotToStringToArraysDotCopyOfRange(TR::Compilation *comp)
```

Comment:
Just a recommendation : You do not need to explicitly write the Dot in the function name.

Reply:
Removed "dot" from the name.